### PR TITLE
feat: Make dotnet version configurable

### PR DESCRIPTION
**Pull request by czthiele**
- add input parameter dotnet_version with default of 6.0.x

### CODE REVIEW

File: .github/workflows/develop.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@ on:
         description: 'If this build should be packed as NuGet package.'
         required: true
         type: boolean
+    dotnet_version:
+        description: 'Define the .NET SDK version the runner should use, e.g. 6.0.x.'
```
Comment (georg-eckert-zeiss) [inline .github/workflows/develop.yml:17]: The .NET SDK version that should be used by the runner (e.g. 6.0.x).

File: .github/workflows/feature-branch.yml [hunk 2]
```diff
@@ -14,6 +14,11 @@ on:
         description: 'If this build should be packed as NuGet package.'
         required: true
         type: boolean
+    dotnet_version:
+        description: 'Define the .NET SDK version the runner should use, e.g. 6.0.x.'
```
Comment (georg-eckert-zeiss) [inline .github/workflows/feature-branch.yml:18]: See other comment

**Review by georg-eckert-zeiss — CHANGES_REQUESTED** [review]
I'd change the wording. Everything else looks fine.

File: .github/workflows/develop.yml (hunk 1 above)
Comment (czthiele) [reply .github/workflows/develop.yml]: Done 👍

File: .github/workflows/feature-branch.yml (hunk 2 above)
Comment (czthiele) [reply .github/workflows/feature-branch.yml]: Done

**Review by georg-eckert-zeiss — APPROVED** [review]

**Pull request merged**